### DISCUSSION TITLE
Feature request: View media files in editor like VS Code

### DISCUSSION THREAD

**Discussion by x4fingers** [zed-industries/zed#9146] [Feature Requests]
This feature should be placed at the top priority of development progress.

I'm a web developer, I always need to open media files (images, videos) to preview in the text editor. So this feature quietly makes my development progress more convenient.

**Comment by bennetbo** [comment]
Sounds like a duplicate of #5251 and #6915. Actually the other two issues could probably be merged into a single one?

**Reply by x4fingers** [reply]
Thanks for showing me, it should be merged.

**Reply by Moshyfawn** [reply]
Done.
